tests/keywords/test_androidUtils.py

```diff
@@ -0,0 +1,60 @@
+import logging
+import sys
+import unittest
+import appium
+import mock
+
+from AppiumLibrary.keywords import _ApplicationManagementKeywords
+from webdriverremotemock import WebdriverRemoteMock
+from AppiumLibrary.keywords import _AndroidUtilsKeywords
+
+
+logger = logging.getLogger()
+stream_handler = logging.StreamHandler(sys.stdout)
+logger.addHandler(stream_handler)
+stream_handler = logging.StreamHandler(sys.stderr)
+logger.addHandler(stream_handler)
+   
+class AndroidUtilsTests(unittest.TestCase):
+    am=None
+    au=None
+
+    def tearDown(self):
+        self.am.close_all_applications()
+
+    def setUp(self):
+        self.am = _ApplicationManagementKeywords()
+        self.am._debug = mock.Mock()
+        # Uncomment to use Mock / Comment to test against real appium instance
+        appium.webdriver.Remote = WebdriverRemoteMock
+        # log debug from _ApplicationManagementKeywords to console
+        self.am._debug = logger.debug
+        self.am.open_application('http://10.1.160.124:4724/wd/hub', alias='MsB1', deviceName='MsB1', udid='d81e91ba', platformVersion='4.4', appPackage='com.android.contacts', platformName='Android', appActivity='.activities.DialtactsActivity')    
```

Comment:
yes, it's now using Mock - if commented it uses the real thing, which I needed to verify if it really works :-)